src/NUnitTestAdapterTests/NavigationDataTests.cs

```diff
@@ -47,6 +47,7 @@ public void TearDown()
         [TestCase("+GenericFixture`2", "Matches", 116, 117, 117, 118)]
         [TestCase("+GenericFixture`2+DoublyNested", "WriteBoth", 132, 133, 134, 134)]
         [TestCase("+GenericFixture`2+DoublyNested`1", "WriteAllThree", 151, 152, 153, 153)]
+        [TestCase("+DerivedClass", "EmptyMethod_ThreeLines", 160, 161, 161, 161)]
```

Comment:
@CharliePoole I see you updated `DerivedClass` but the method is still `EmptyMethod_ThreeLines` here and `EmptyMethodThreeLines` in `BaseClass`. That is incorrect isn't it?